src/components/Sidebar/user/Controls.tsx

```diff
@@ -48,15 +48,17 @@ export const useLegacyThemeHandler = () => {
 export const UserControls = () => {
   const { user, logout } = useUser();
   const { changeTheme, theme } = useLegacyThemeHandler();
+  const [open, setOpen] = useState(false);
 
   return (
-    <Dropdown>
+    <Dropdown open={open} onOpenChange={value => !value}>
       <Dropdown.Trigger>
         <Button
           variant="tertiary"
           icon={<MoreOptionsIcon />}
           data-test-id="userMenu"
           size="medium"
+          onClick={() => setOpen(true)}
```

Comment:
**nitpick** this line is not required I think - radix trigger automatically handles the event but i can be left as it is